src/client/pfs_file.go

```diff
@@ -174,35 +160,36 @@ func (mfc *modifyFileCore) maybeError(f func() error) (retErr error) {
 	return f()
 }
 
-func (mfc *modifyFileCore) sendAppendFile(req *pfs.AppendFile) error {
+func (mfc *modifyFileCore) sendPutFile(req *pfs.PutFile) error {
 	return mfc.client.Send(&pfs.ModifyFileRequest{
-		Modification: &pfs.ModifyFileRequest_AppendFile{
-			AppendFile: req,
+		Modification: &pfs.ModifyFileRequest_PutFile{
+			PutFile: req,
 		},
 	})
 }
 
-// AppendFileTar appends a set of files from a tar stream.
-func (mfc *modifyFileCore) AppendFileTar(overwrite bool, r io.Reader, tag ...string) error {
+var errV1NotImplemented = errors.Errorf("V1 method not implemented")
```

Comment:
See above, I think we should just break the API.  This was useful when v1 and v2 lived in the same codebase.